lib/rides/models/search_helper.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:intl/intl.dart';
```

Comment:
wollen wir die Klasse in util verschieben wenn sie nur diese beiden Methoden hat, weil die Methoden werden ja an einigen stellen gebraucht dann kann man die Methoden einfach immer daraus nehmen. Oder wir machen nen Interface draus weis nicht wie gut das in Dart geht. 
Könnte man auch für die Date und TimePicker bzw. Validator machen die haben wir ja auch mehrmals.

Reply:
hast recht